Add comment to feature overview (#244)
AY2122S1-CS2103T-W16-2/tp: docs/UserGuide.md

* Client bookmarks are sorted in ascending alphabetical order of client name.
  Uppercase letters appear before lowercase letters.

> * All Client commands are able to function properly when entered from any tab.

Reviewer: Can't we say something like 
```suggestion
* You can use all client commands from any tab.
```
